lib/insights/api/common/filter.rb

```diff
@@ -214,63 +212,63 @@ def arel_lower(key)
         def comparator_contains(key, value)
           return value.each { |v| comparator_contains(key, v) } if value.kind_of?(Array)
 
-          self.query = query.where(model_arel_attribute(key).matches("%#{query.sanitize_sql_like(value)}%", nil, true))
+          @query = query.where(model_arel_attribute(key).matches("%#{query.sanitize_sql_like(value)}%", nil, true))
```

Comment:
Should we also define a setter method so that these don't need to change?
```ruby
def query=(new)
  @query = new
end
```

Reply:
👍 with one minor update, Rubocop preferred having the attr_writer instead, works the same and we don't need to change the self.query = ...